src/index.ts

```diff
@@ -9,20 +9,19 @@ export {
   Authorize,
   AuthorizeSourceData,
   AuthorizeResult,
-  AuthorizationError,
   ActionConstraints,
   LogLevel,
   Logger,
 } from './App';
 
-export { ErrorCode } from './errors';
-
 export {
   default as ExpressReceiver,
   ExpressReceiverOptions,
 } from './ExpressReceiver';
 
+export * from './errors';
 export * from './middleware/builtin';
+export * from './receiver';
```

Comment:
We should remove this export if the plan is to release `receiver.ts` as its own module.